config/rbac/role.yaml

```diff
@@ -223,4 +223,19 @@ rules:
   - delete
   - watch
   - patch
+- apiGroups:
+  - operator.openshift.io
+  resources:
+  - imagecontentsourcepolicies
+  verbs:
+  - get
+  - create
```

Comment:
Maybe add update as well in case there is a change during an update of the chart? Do we need watch? We will need delete to remove it when the CR is deleted. What do you think?

Reply:
Just to be safe I added
```
  - update
  - delete
 ```